### PR TITLE
updated docs to rename the binary per the os system you are on

### PR DESCRIPTION
**Pull request by ValClarkson**
Issue: [16700]

### CODE REVIEW

File: docs/content/_index.md [hunk 1]
```diff
@@ -28,14 +28,19 @@ The `kubectl-pgo` binary is available either through the Crunchy Data [Access Po
 
 ## Installing the Client
 
-Once downloaded, move the `kubectl-pgo` binary to `/usr/local/bin` and make it executable by running
-the following commands:
+### Linux or Mac
+
+If you downloaded the Linux binary rename the binary from `kubectl-pgo-linux-amd64` to `kubectl-pgo` or if you download the Mac binary rename the binary from `kubectl-pgo-darwin-amd64` to `kubectl-pgo` then move the `kubectl-pgo` binary to `/usr/local/bin` and make it executable by running the following commands:
```
Comment (dsessler7) [inline docs/content/_index.md:33]: :pick:

```suggestion
If you downloaded the Linux binary, rename the binary from `kubectl-pgo-linux-amd64` to `kubectl-pgo`. If you downloaded the Mac binary, rename the binary from `kubectl-pgo-darwin-amd64` to `kubectl-pgo`. Once renamed, move the `kubectl-pgo` binary to `/usr/local/bin`, and make it executable by running the following commands:
```

File: docs/content/_index.md [hunk 2]
```diff
@@ -28,14 +28,19 @@ The `kubectl-pgo` binary is available either through the Crunchy Data [Access Po
...
 
 ```
 sudo mv /PATH/TO/kubectl-pgo /usr/local/bin/kubectl-pgo
 sudo chmod +x /usr/local/bin/kubectl-pgo
 ```
 
+### Windows
+
+If you downloaded the Windows binary rename `kubectl-pgo-windows-386` to `kubectl-pgo.exe` this allows Windows to recognize the file as an executable.  From there open a power shell terminal in the same directory to execute the binary.
```
Comment (dsessler7) [inline docs/content/_index.md:42]: :pick: 
```suggestion
If you downloaded the Windows binary, rename `kubectl-pgo-windows-386` to `kubectl-pgo.exe`. This allows Windows to recognize the file as an executable.  From there, open a power shell terminal in the same directory to execute the binary.
```

:thinking: Is opening a terminal necessary? Are we prompting the user to just leave the terminal open or to actually execute the binary?

Comment (ValClarkson) [reply docs/content/_index.md:42]: It was in the notes in the story of what a customer had to do.  I can remove it, I'm not on windows so I wasn't sure if it was needed or not

**Review by dsessler7 — APPROVED** [review]
LGTM

**Pull request merged**